test/lookup_test.py

```diff
@@ -10,12 +10,12 @@
 def test_persistent_object(servicer, client):
     volume_id = Volume.create_deployed("my-volume", client=client)
 
-    v: Volume = Volume.lookup("my-volume", client=client)
+    v = Volume.from_name("my-volume").hydrate(client)
     assert isinstance(v, Volume)
```

Comment:
unrelated to the PR, but this isinstance check can't fail. I think they are remnants of when lookups could return any object type

Reply:
when lookups could do _what_ lol